config/bidderinfo_test.go

```diff
@@ -38,6 +39,9 @@ experiment:
   adsCert:
     enabled: true
 endpointCompression: "GZIP"
+openrtb:
+  version: "2.6"
```

Comment:
Nitpick: The quotes aren't necessary, and we might want to prefer the simple style of `version: 2.6`.

Reply:
Should I get rid of the quotes on the other values above?

Reply:
Sounds like a good idea to keep things simple.